src/main/java/me/coley/cafedude/io/AttributeReader.java

```diff
@@ -551,4 +555,145 @@ private ConstantValueAttribute readConstantValue() throws IOException {
 		int valueIndex = is.readUnsignedShort();
 		return new ConstantValueAttribute(nameIndex, valueIndex);
 	}
+
+	private TypeInfo readVerificationTypeInfo() throws IOException {
+		// u1 tag
+		int tag = is.readUnsignedByte();
+		switch (tag) {
+			// Top_variable_info
+			case 0:
```

Comment:
For legibility instead of having comments here breaking up the switch, can add some values to the constants class and use them instead of literals.